pkg/detectors/ramp/ramp_integration_test.go

```diff
@@ -0,0 +1,164 @@
+//go:build detectors
+// +build detectors
+
+package ramp
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
```

Comment:
nit: Could we move this import down one line so it's grouped with the others.

Reply:
Thanks for pointing that out. I went in and fixed the imports for all the new integration tests.